### DISCUSSION TITLE
Can't seem to collapse enchantment books

### DISCUSSION THREAD

**Discussion by MasterGaro** [shedaniel/RoughlyEnoughItems#1330] [Q & A] [answered]
I'm trying to select all the enchantment books and collapse them, but it doesn't work. I even tried just doing Protection 1 to 8 (All The Mods 8) to see if worked on just one variant of an enchant. Still didn't work. Am I doing something wrong?

**Comment by shedaniel** [comment, marked as answer]
You don't really need to select them to collapse them? Just Alt-Click (Windows, Linux) or Option-Click (macOS)
